app/src/main/java/org/mifos/selfserviceapp/ui/fragments/AccountsFragment.java

```diff
@@ -184,7 +184,7 @@ public void showError(String errorMessage) {
 
     @Override
     public void showProgress() {
-        showSwipeRefreshLayout(true);
+
```

Comment:
Revert back this change, it is the swipe refresh layout that was showing on loading the any type of accounts.